### ISSUE TITLE
Determine approach for maintaining Design assets, and how Designers contribute to the project

### ISSUE THREAD

**Issue by PaulaPaul** [finos/a11y-theme-builder#844]
## Problem/Concern

Theme Builder design assets were created and maintained in Figma, under a license outside of the control of FINOS. Now that the project has been donated to FINOS we would like to have a sustainable way for design assets to stay associated with the project in the FINOS GitHub (for example, links to design assets in another organization's Figma may change ownership, and can make it challenging to collaborate on and modify designs).

## Proposed Solution

There are multiple potential solutions to creating a FINOS repository for design artifacts that can be a long term home for the Theme Builder project. The goal of this issue is to discuss alternatives with the project team and document the approach for contributing and maintaining design assets associated with Theme Builder.

Along the way we may uncover and share best practices for incorporating design assets in open 
source projects!

### Alternatives

- FINOS Sponsored Figma subscription: Pro subscription supporting up to four designers, which can be managed by the core contributors to Theme Builder
- [Penpot](https://penpot.app/) is open source, and recently added a capability to import Figma designs. This is related to #665 
- Other alternatives? Also, this discussion will help us with 'end to end accessibility' and how to deliver that

For any alternative, we have been asked to summarize the approach and costs (if any) for FINOS consideration, ideally with the idea that this approach may apply to other open source projects or open source design efforts in general.

**Comment by aaronreed708** [comment]
Waiting for Figma licenses decision to resolve before closing this issue with that update

**Comment by PaulaPaul** [comment]
Sent this message to @maoo, @lwnoble and @aaronreed708 today for consideration - comments are welcome here as well from the community!

> Hi Mau,
> 
> I had a good discussion with the folks at Figma last week and wanted to summarize for you, Aaron, and Lise with a recommendation, for your thoughts. Context, alternatives, considerations, and next steps are below-
> 
> Context:
> - Discover Financial donated the Accessibility Theme Builder to FINOS in 2023 (wow! things have come a long way pretty quickly!)
> - In addition to the code and content in GitHub, the project's core design assets are maintained in Figma, which is common in the industry
> - Currently, the design artifacts are kept in Discover's Figma environment, and ideally these should be moved to a FINOS managed environment to sustain the project going forward under FINOS stewardship
> 
> Alternatives considered:
> - Alternatives to Figma have been considered, including [Penpot](https://penpot.app/) as the most likely open source alternative.
> There would be a loss of fidelity and capability if the Theme Builder team moved the design assets to Penpot. Also, Theme Builder currently relies on advanced Figma features that are only available with a Figma Enterprise seat.
> - Paula spoke with Figma about non-profit foundation licensing for Figma, and how we might address enabling FINOS stewardship of the design assets in Figma. Notes:
> - Figma provides 'seat by seat' licenses, and does not offer specialized licensing or discounts for nonprofits. 
> - For invoicing/PO purchases, there is a minimum annual cost that is covered by Figma's proposal (below) but it was unclear how negotiable that minimum may be.
> - Figma suggested a solution (copy here from the email below) with annual invoicing to FINOS.
> 5 Design, 5 FigJam, and 5 DevMode seats would be $6,900 annually (you can switch/reassign these seats whenever needs change internally)
> Sales-assisted demos for the team
> Invoice that can be paid via wire transfer, ACH, or CC, payment terms are net-30
> Minimum contract value can be met through FigJam, DevMode, and Figma Design seats, so we can work together to best meet your needs if budget falls below $6.9k
> - This licensing can be shared across all FINOS projects and managed centrally.
> 
> Considerations:
> - Instead of offering a discount or 'free licenses' Figma suggested working with FINOS for flexibility with the annual true up to optimize spend if the licenses go up or down over the year. 
> - In addition, Figma asked about event sponsorship opportunities, and would be open to  sponsoring a second LF Mentorship for Theme Builder (for ~6k, or potentially less depending on geography), for a design-focused mentee.
> 
> Recommendation:
> - If it is within the FINOS budget for 2024, negotiate with Figma for pro-rated 2024 licensing, starting July 1 (half year), with a true up date of January 1 2025.
> - Best case: Negotiate better options than the two below...
> - Ok/Good case: pro-rated 2024 invoice for $3,450 on Jul 1, 2024 (due Aug 1), explore multi-year contract options. Yearly true up on Jan 1 2025 and beyond for $6,900.
> - Worst case: $6,900 invoice on Jul 1 2024 (due Aug 1), with true up invoice on Jul 1 2025 and beyond
> - Ask Figma to sponsor a ~6000K LF Membership for Theme Builder, to focus on a design mentee, with cross promotion. 
> - Plan to record and share education demos with Figma and the FINOS Theme Builder team.
> - Share sponsorship opportunities for OSFF (London and NYC)
> 
> Next steps:
> I have a follow up call with Figma next week, on Wed May 29th at 11am ET, and can add any and all to that discussion - will ping you all on Slack for thoughts.
> 
> Thanks for your time and ideas!

**Comment by PaulaPaul** [comment]
@lwnoble and @aaronreed708 - the discussions with FINOS about the direction for managing design assets for FINOS Projects are ongoing, and there are some options. The statements below are just ideas for consideration-

We could potentially host PenPot (as penpot.finos.org) so that Theme Builder and other FINOS projects can use it for design assets. We know PenPot can't do what the current commercial software does, but rather than paying for commercial licenses, we could potentially work with FINOS to open up another mentorship specifically for design and to begin the (long) process of moving design assets to FINOS' PenPot instance while contributing to PenPot improvements along the way.  That is a very long term solution so while we are in transition we would need to keep supporting the project with existing commercial licenses, and we would need to start finding ways to move design assets to PenPot (and opening issues against PenPot for features we need).

This is a path to getting another Theme Builder mentorship specifically for Design - 

I've copied part of my conversation thread on this below, and would like to discuss this at the project call this week.

---
The original request for commercial licenses came from the team since there is currently no way to maintain or improve the design assets (or grow the community of contributors) for Theme Builder outside of commercial licenses not under FINOS control, so it's a challenge. 

But, we also love the idea of moving the design assets to an open source solution eventually - the availability of open source design solutions for accessible design is a great area for community contribution.

If we could get confirmation of investment in mentorship and hosting PenPot in FINOS cloud infrastructure, the team could get behind efforts to move the assets to PenPot over time. That would also solve the issue of FINOS ownership of the design assets.

Since PenPot is a self-hosted solution, what do you think of the following proposal:

- FINOS will let the Theme Builder team host PenPot in the FINOS sandbox or FINOS cloud infrastructure, so they can begin to transfer design assets there. This may take some time, given the current capability gaps in PenPot.

- FINOS will fund another Theme Builder mentorship specifically for Design, aiming to improve PenPot capabilities and move the Theme Builder assets to PenPot.

- This mentorship could be announced at OSFF, where Lise will be speaking.

- During the transition to PenPot, we would still need commercial license support for Theme Builder assets and support the goal of moving the assets to the FINOS PenPot instance.

Either way, investment is needed, but I like this option much better as a step towards making the project sustainable as an independent FINOS open-source project.

Additional potential benefits:

- Other FINOS projects could take advantage of the FINOS-hosted PenPot instance for their design assets (and also contribute to the project)
- We could encourage PenPot to donate the project to FINOS or the Linux Foundation (similar to OpenTofu) since robust open-source design tooling for accessibility is a need across the industry

- FINOS could publish unique content and thought leadership on the importance of open design tooling and accessibility.

**Comment by aaronreed708** [comment]
> he is working with FINOS to open up another mentorship specifically for design and to begin the (long) process of moving design assets to FINOS' PenPot

I would say that we need to evaluate PetPot's shortcomings, prioritize addressing them, and pick the mentorship based on that.  Which I assume everyone can agree with.  I just don't want to say that this will be a design mentorship until we know better.  I assume that the competitive advantages that Figma currently enjoys are largely functional.  But I may be wrong, I am certainly not strong  in the areas of design tooling.

**Comment by PaulaPaul** [comment]
Agreed @aaronreed708 - I think our first step is to establish (with FINOS) the long term direction for design assets (for example, 'FINOS wishes to ensure the theme builder is self-sustaining as an open source project and does not rely on specific organizations or commercial tooling'). Once we have an agreement or a statement on the long-term direction, then we can start shifting in that direction over time. In the meantime, I think it's a great idea to have stories on the board related to evaluating PenPot and understanding the relatively new plug in that would allow us to import existing design assets (and, open issues against PenPot for gaps in the import process). Once we have a good import process, we can start to open issues related to features we'd like to see.
